Sources/MessagingPush/Store/PushHistory.swift

```diff
@@ -0,0 +1,47 @@
+import CioInternalCommon
+import Foundation
+
+protocol PushHistory: AutoMockable {
+    func hasHandledPushClick(deliveryId: String) -> Bool
+    func handledPushClick(deliveryId: String)
+}
+
+// sourcery: InjectRegister = "PushHistory"
+class PushHistoryImpl: PushHistory {
+    private let keyValueStorage: KeyValueStorage
+
+    var numberOfPushesToTrack = 100
```

Comment:
What does this value represent? I think adding a comment would add value to it.

Reply:
Thanks for asking. I changed the name to `maxSizeOfHistory` to be more descriptive of it's purpose.